frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1687,15 +1688,16 @@ function ReaderHighlight:deleteHighlight(page, i, bookmark_item)
         logger.dbg("delete highlight from document", removed)
         self.ui.document:deleteHighlight(page, removed)
     end
+    UIManager:setDirty(self.dialog, "ui")
```

Comment:
(Just asking: why wasn't it needed before , why it is needed now ?)

Reply:
It's a fix, follow up to https://github.com/koreader/koreader/pull/8600 "_(5) do not save highlight if "Add note" is cancelled_".
Need to clean up the highlight if Add note has been cancelled.